supportedBackends/sfs/src/main/scala/cromwell/backend/sfs/SharedFileSystem.scala

```diff
@@ -73,19 +82,41 @@ trait SharedFileSystem extends PathFactory {
 
   def sharedFileSystemConfig: Config
 
-  lazy val LocalizationStrategies = sharedFileSystemConfig.getStringList("localization").asScala
-  lazy val Localizers = localizePathAlreadyLocalized _ +: (LocalizationStrategies map {
-    case "hard-link" => localizePathViaHardLink _
-    case "soft-link" => localizePathViaSymbolicLink _
-    case "copy" => localizePathViaCopy _
-    case unsupported => throw new UnsupportedOperationException(s"Localization strategy $unsupported is not recognized")
-  })
+  lazy val DefaultStrategies = Seq("hard-link", "soft-link", "copy")
+
+  lazy val LocalizationStrategies = getConfigStrategies("localization")
+  lazy val Localizers = createStrategies(LocalizationStrategies, docker = false)
+  lazy val DockerLocalizers = createStrategies(LocalizationStrategies, docker = true)
+
+  lazy val CachingStrategies = getConfigStrategies("caching")
+  lazy val Cachers = createStrategies(CachingStrategies, docker = false)
+
+  private def getConfigStrategies(configPath: String): Seq[String] = {
+    if (sharedFileSystemConfig.hasPath(configPath)) {
+      sharedFileSystemConfig.getStringList(configPath).asScala
+    } else {
+      DefaultStrategies
+    }
+  }
+
+  private def createStrategies(configStrategies: Seq[String], docker: Boolean): Seq[DuplicationStrategy] = {
+    // If localizing for a docker job, remove soft-link as an option
+    val filteredConfigStrategies = configStrategies filter {
+      case "soft-link" if docker => false
+      case _ => true
+    }
 
-  // Note that any unrecognized configuration will be raised when Localizers (just above) gets resolved.
-  lazy val DockerLocalizers = localizePathAlreadyLocalized _ +: (LocalizationStrategies collect {
-    case "hard-link" => localizePathViaHardLink _
-    case "copy" => localizePathViaCopy _
-  })
+    // Convert the (remaining) config strategies to duplication strategies
+    val mappedDuplicationStrategies = filteredConfigStrategies map {
```

Comment:
This method gets passed a string of configurations. The method is called with a separate seq of configs by the execution (aka localization) and call caching. So one should be able to separate the two.